src/lib/bootloader/grub2_widgets.rb

```diff
@@ -359,8 +359,10 @@ def help
           "<p><b>Secure Boot Support</b> if checked enables Secure Boot support.<br>" \
           "This does not turn on secure booting. " \
           "It only switches to the new secure-boot enabled boot data format. " \
-          "Note that this new format works only on z15 or later and only for zFCP disks. " \
-          "The system does not boot if these requirements are not met.</p>"
+          "Note that this new format works only on z15 or later and "\
+          "disk type supports depends on hardware. " \
+          "For details consult requirements at " \
+          "https://www.ibm.com/docs/en/linux-on-systems?topic=introduction-requirements</p>"
```

Comment:
I would rephase it a bit:

```suggestion
          "Note that this new format works only on z15 or later and "\
          "only for some types of disk. The supported disks depends on " \
          "the hardware. For more details see the requirements at " \
          "https://www.ibm.com/docs/en/linux-on-systems?topic=introduction-requirements</p>"
```